corehq/apps/export/tests/test_export_data_schema.py

```diff
@@ -539,9 +539,12 @@ def setUpClass(cls):
         cls.first_build.copy_of = cls.current_app.get_id
         cls.first_build.version = 3
 
+        cls.advanced_app = Application.wrap(cls.get_json('advanced_application'))
```

Comment:
👍 originally was doing it because i also needed to define the form xml. agree it's better to define the app programmatically for a variety of reasons. will update